avocado/core/task/runtime.py

```diff
@@ -46,8 +46,10 @@ def __init__(self, task):
         """
         #: The :class:`avocado.core.nrunner.Task`
         self.task = task
-        #: Additional descriptive information about the task status
+        #: A :class:`avocado.core.task.runtime.RuntimeTaskStatus` value
         self.status = None
+        #: Additional descriptive information about the task status
+        self.status_description = None
```

Comment:
I agree with you that usage of `RuntimeTask.status` is wrong in `podman_spawner`, but IMO the `self.status_description` is not necessary.  In podman spawner, I would only log the errors without changing the status.